test/e2e-cypress/e2e/features/oas3-request-body-required.cy.js

```diff
@@ -79,6 +79,7 @@ describe("OpenAPI 3.0 Validation for Required Request Body and Request Body Fiel
         )
         .should("not.have.class", "invalid")
         .clear()
+        .blur() // HACK: force the debounce on the textarea to trigger
```

Comment:
> I had to modify a few tests to call blur() on the body to force the debounce to trigger. I would have expected the tests to work without having to call blur() however I don't know enough about Cypress to know if this is normal.

I'd remove the comments. It's not really a hack. The debounced textarea will call the handler in following situations:

- `debounceTimeout` expires
- focus leaves the textarea (blur) - enabled by default by `forceNotifyOnBlur` option

Now because cypress is automated testing, things are happening fast. `debounceTimeout` is not yet expired, when we want to check for assertions. So we trigger the blur so that we're sure the handler has already been called before we continue with assertions.